package rec.widget.tree;

message NavigatorConfig {
    optional bool allow_file_drop = 1 [default = false];
    optional bool expand_to_fit_window = 2 [default = true];
    optional bool contract_to_fit_window = 3 [default = true];
};
